tests/resources/setup.js

```diff
@@ -95,6 +95,21 @@ export function createSoldItemNet(Item) {
   })
 }
 
+/**
+ * Create sold item with net price 0
+ * @return {Item}
+ */
+export function createSoldItemNetZero(Item) {
```

Comment:
please delete it. we don't use it anywhere. Just revert the file to prevent extra new lines and white spaces

Reply:
also have to remove the function name from the imports of the item.js

thx